about.qmd

```diff
@@ -73,7 +75,8 @@ If you would like to see a blog post on a particular topic, submit an issue on o
 - [X] [Cell typing: what we've found to work](posts/cell-typing-basics/index.qmd)
```

Comment:
Could you please add a link to the 2 cell profile repos to the last bullet point of InSituType section?

Reply:
Links to both repos have been added to the general cell typing post